tests/react/vanilla-utils/atomWithObservable.test.tsx

```diff
@@ -408,7 +367,9 @@ it('synchronous subscription with BehaviorSubject', async () => {
 
   const { findByText } = render(
     <StrictMode>
-      <Counter />
+      <Suspense fallback="loading">
```

Comment:
Idk why, but this is needed when testing against React 17, which is used by the CI work flow.